tests/storage/test_user_directory.py

```diff
@@ -696,6 +696,8 @@ def test_icu_word_boundary_punctuation(self) -> None:
                 ["lazy'fox", "jumped", "over", "the", "dog"],
                 # ICU 70 on Ubuntu 22.04
                 ["lazy'fox", "jumped:over", "the.dog"],
+                # ICU 2.10.2 on Alpine edge / macOS
```

Comment:
This looks like a pyicu version, not a libicu version like the other two above.

Reply:
@DMRobertson done